### PR TITLE
MGMT-1015 Fix date format in Swagger APIs

### PR DESCRIPTION
**Pull request by empovit**
This PR is pending UI changes. Otherwise errors invalid format will be shown instead of dates in the UI.

### CODE REVIEW

File: cmd/main.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ import (
 )
 
 func init() {
-	strfmt.MarshalFormat = strfmt.ISO8601LocalTime
+	strfmt.MarshalFormat = strfmt.RFC3339Millis
```
Comment (filanov) [inline cmd/main.go:55]: Does posgres support this format?

Comment (empovit) [reply cmd/main.go:55]: Here's an example of what's being returned by gorm `CreatedAt:2020-09-13T06:26:46.000Z`, before being marshalled by Swagger. The JSON looks like `{ "created_at": "2020-09-13T06:26:46.000Z" }`. In any case, I've tested this change E2E including the UI.

Comment (filanov) [reply cmd/main.go:55]: So what is the difference?

Comment (empovit) [reply cmd/main.go:55]: Compare with `strfmt.ISO8601LocalTime`, where Swagger drops the UTC designation making it a local time `{"created_at": "2020-09-13T06:26:46"}`.

Comment (empovit) [reply cmd/main.go:55]: > So what is the difference?

Local time vs explicitly specifying UTC offset https://en.wikipedia.org/wiki/ISO_8601

Comment (filanov) [reply cmd/main.go:55]: So the change is how the time returned from swagger?

Comment (empovit) [reply cmd/main.go:55]: Correct. See https://github.com/mareklibra/facet-lib/pull/198 and MGMT-1015.

**Review by filanov — APPROVED** [review]

**Comment by empovit** [comment]
Pending https://github.com/openshift-metal3/facet/pull/302

**Pull request merged**